### PR TITLE
allow equality to avoid flaky on clock

### PR DESCRIPTION
**Pull request by zhouhaibing089**
Fixes https://github.com/kubernetes/kubernetes/issues/24784.

/cc @yujuhong

### CODE REVIEW

File: pkg/kubelet/image_manager.go [hunk 1]
```diff
@@ -267,7 +267,7 @@ func (im *realImageManager) freeSpace(bytesToFree int64, freeTime time.Time) (in
 	spaceFreed := int64(0)
 	for _, image := range images {
 		// Images that are currently in used were given a newer lastUsed.
-		if image.lastUsed.After(freeTime) {
+		if freeTime.Before(image.lastUsed) {
```
Comment (yujuhong) [inline pkg/kubelet/image_manager.go]: This doesn't allow equality. Let's make the comparison explicit.
```
if image.lastUsed.Equal(freeTime) || image.lastUsed.After(freeTime) {
```

Comment (zhouhaibing089) [inline pkg/kubelet/image_manager.go]: sorry, my mistake.

**Comment by zhouhaibing089** [comment]
@yujuhong addressed as you suggested. Thanks.

**Comment by yujuhong** [comment]
LGTM. We should add a unit test to for this case specifically. This requires using the fake clock (#24959). Since the PR is still inflight, we can tackle this later.

**Pull request merged**